### PR TITLE
Use github pretty code formatting

### PR DESCRIPTION
**Pull request by luispabon**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 /vendor
-/codeCoverage
\ No newline at end of file
+/codeCoverage
+.idea
```
Comment (TomChaton) [inline .gitignore:3]: Is it traditional to include IDE specific project files?
I've just put mine in .git/info/exclude to try and keep this clean

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 [![Build Status](https://travis-ci.org/tomcroft/tantrum_elastic.svg?branch=dev)](https://travis-ci.org/tomcroft/tantrum_elastic)[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/tomcroft/tantrum_elastic/badges/quality-score.png?b=dev)](https://scrutinizer-ci.com/g/tomcroft/tantrum_elastic/?branch=dev)[![Coverage Status](https://coveralls.io/repos/tomcroft/tantrum_elastic/badge.svg?branch=dev&service=github)](https://coveralls.io/github/tomcroft/tantrum_elastic?branch=dev)
 
 ## Example Low Level usage
-
+```php
```
Comment (TomChaton) [inline README.md:7]: :muscle:

File: .gitignore (hunk 1 above)
Comment (luispabon) [inline .gitignore:3]: Yes, I've seen it in quite a few places for phpstorm, netbeans etc as the project folders are usually stored within the source tree for portability

**Comment by TomChaton** [comment]
More than happy to merge. Could you open it against dev, though please?

**Comment by luispabon** [comment]
Closed in favour of https://github.com/tomcroft/tantrum_elastic/pull/87

**Pull request closed without merging**